src/gulpfile.babel.js

```diff
@@ -363,3 +368,28 @@ gulp.task(
 		gulp.watch( config.imgSRC, gulp.series( 'images', reload ) ); // Reload on customJS file changes.
 	})
 );
+
+/**
+ * Generate .zip
+ *
+ * Generate my-theme-folder.zip in your `/wp-content/themes` folder.
```

Comment:
WP Gulp is used by both themes and plugins. Adding something just for themes is not right. Care to test for both and adjust the lingo?

Reply:
Okay Got it I will test it on Plugin also and then send again pull request.

Reply:
Sure, looking forward.